packages/react-native-windows-init/src/Cli.ts

```diff
@@ -427,14 +427,6 @@ function isProjectUsingYarn(cwd: string): boolean {
       );
     }
 
-    if (argv.useHermes && argv.experimentalNuGetDependency) {
```

Comment:
should this block remain for 64 and lower?

Reply:
FWIW we don't publish the copy of the package from this branch at all

Reply:
We really ought to do the CLI work to couple CLI args to the per-version generator

Reply:
@NickGerleman  yes sorry I meant when this is ported to main, we should do something to keep blocking it on 64 and below

Reply:
IMHO the time is better spent on the CLI flags architecture fix in 0.66+ than to try to make react-native-windows-init detect and special case by version. Someone creating a new project is most likely to target latest, and this isn't the first time we've seen the issue. E.g. the community module flag does nothing on versions before 0.64. 

I've kept a bit quiet, but I did actually add a C# check when I initially added the --useHermes flag. I fixed one of the C# issues and removed the check in the PR, but discovered we still had the NuGet package issue. I left the change to remove the warning though. The rationale was that if the CLI tells someone the combination isn't supported, they will just avoid Hermes. If they get a build error, they are more likely to treat the issue as a bug (which it is), and give us signal the scenario is important.

Reply:
I might try to make the CLI architecture fix tonight. I really don't think there is that much work, but this problem seems to keep on coming up.